src/frontend/css/style.css

```diff
@@ -3,16 +3,64 @@
   padding: 0;
   box-sizing: border-box;
 }
+body {
+  /*background-color: rgb(121, 197, 150);*/
+  background-color: rgb(109,194,202);
+}
+h1 {
+  font-family: "Century Gothic", Courier, monospace;
```

Comment:
I think century gothic is a windows font. I don't have it on my linux system. Everything falls back to courier for me. Might want to choose a different font. You can use a font from google fonts if you want: https://fonts.google.com/

Reply:
I swapped century gothic out with the BIZ UDPGothic google font. The submit button is also animated now.